### DISCUSSION TITLE
Input value limitation before decimal place

### DISCUSSION THREAD

**Discussion by akashramtohul4** [quasarframework/quasar#12913] [CLI - PWA mode]
Hello friends

I want to limit the number of values before a decimal place. 
For example :
 I can input any number after the decimal place 
like this **123456.111** but before the decimal place I can add only **3 numbers** 

Am trying to solve the problem using mask but am unable to get the solution 

https://codepen.io/akashramtohul4/pen/vYpyQpZ

Can someone please help 

Thanks

**Comment by pdanpdan** [comment]
https://codepen.io/pdanpdan/pen/MWrbzzg?editors=1010
use `fill-mask` to have something before the decimal point
